cluster/prometheus.bash

```diff
@@ -40,6 +40,11 @@ function RUN_GPU_CHECK {
 	return 0
 }
 MAINQ_ASK="no"
-MODULES_ADD_DEF="apps/r"
+
+MODULES_ADD_DEF="apps/r"  
+# Prometheus's module apps/r loads apps/cuda/9.0 as dependency. 
+# Moreover apps/r loads tools/gcc/6.4.0 which causes conflict with openmpi/2.0.1-gcc-4.9.2, 
+# thus apps/r (with its dependencies) must be called first $MODULES_ADD, then override with $MODULES_RUN
+
```

Comment:
There is no need to use gcc-4.9.2
this
`
        module load apps/r/3.4.4
        module load plgrid/apps/cuda/9.0
        module load plgrid/tools/openmpi/2.1.1-gcc-6.4.0
        module load libs/python-numpy/1.14.2-python-2.7


`
works ok. 
Please check that with config script and reupload